airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/spec.json

```diff
@@ -0,0 +1,30 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/amazon-ads",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Amazon Ads Spec",
+    "type": "object",
+    "required": ["client_id", "client_secret", "scope", "refresh_token"],
+    "additionalProperties": false,
+    "properties": {
+      "client_id": {
+        "type": "string",
+        "description": "Oauth client id <a href=\"https://advertising.amazon.com/API/docs/en-us/setting-up/step-1-create-lwa-app\">How to create your Login with Amazon</a>"
+      },
+      "client_secret": {
+        "type": "string",
+        "description": "Oauth client secret <a href=\"https://advertising.amazon.com/API/docs/en-us/setting-up/step-1-create-lwa-app\">How to create your Login with Amazon</a>",
+        "airbyte_secret": true
+      },
+      "scope": {
```

Comment:
Please add `"examples": ["..."]` section here with examples of scopes. Like `"examples": ["advertising::campaign_management"]`.